app/inc/http_basic_authen.php

```diff
@@ -61,7 +61,7 @@ function makeExceptionReport($value): void
     $userGroup = !empty($settings["data"]->userGroups->$subUser) ? $settings["data"]->userGroups->$subUser : null;
 
     if ($dbSplit[0] != $schema) {
-        $sql = "SELECT * FROM settings.geometry_columns_view WHERE _key_ LIKE :schema";
+        $sql = "SELECT privileges FROM settings.geometry_columns_join WHERE _key_ = :schema";
```

Comment:
How can you just change operator from '=' to 'LIKE'? And be careful using geometry_columns_join directly, because it may contain stall records

Reply:
we changed to using _join directly because we have installations with upwards of 900 layer in the database, of which a great deal is imported from elsewhere on the server as FT.

the latency added by having postgis validate each layer, local and foreign caused timeouts, increased cpu usage and blocked database sessions - ultimately degrading stack performance and taking the service offline.

going with = is a mistake, i fixed it.

we plan on doing regular maintainance sweepd for stale records in _join